lib/fastlane/plugin/appcenter/actions/appcenter_upload_action.rb

```diff
@@ -50,15 +51,15 @@ def self.run_dsym_upload(params)
             file_name = File.basename(dsym_path)
             dsym_upload_details = Helper::AppcenterHelper.create_mapping_upload(api_token, owner_name, app_name, file_name ,build_number, version)
           else
-            dsym_upload_details = Helper::AppcenterHelper.create_dsym_upload(api_token, owner_name, app_name)
+            dsym_upload_details = Helper::AppcenterHelper.create_dsym_upload(api_token, owner_name, app_name, timeout)
```

Comment:
This only creates an upload URL & ID, so timeout due to large files is not going to happen here.